lib/motorcontrollers/can_processor.cpp

```diff
@@ -0,0 +1,22 @@
+#include "can_processor.hpp"
+
+namespace hyped::motorcontrollers {
+    
+CanProcessor::CanProcessor()
+{
+    //TODO implement
+}
+
+bool CanProcessor::sendMessage()
+{
+    //TODO implement
+    return false;
+}
+
+void CanProcessor::processMessage(core::CanFrame frame)
```

Comment:
No idea how  this area works in general but can `frame` be `const`? Is the frame actually modified in the function or is it just read in and used?